code/game/objects/items/cards_ids.dm

```diff
@@ -935,10 +935,11 @@
 	if(!SSdbcore.Connect())
 		return
 
-	var/intern_threshold = (CONFIG_GET(number/use_low_living_hour_intern_hours) * 60) || (CONFIG_GET(number/use_exp_restrictions_heads_hours) * 60) || INTERN_THRESHOLD_FALLBACK_HOURS * 60
-	var/playtime = user.client.get_exp_living(pure_numeric = TRUE)
+	var/intern_threshold = (CONFIG_GET(number/use_low_living_hour_intern_hours) HOURS) || (CONFIG_GET(number/use_exp_restrictions_heads_hours) HOURS) || INTERN_THRESHOLD_FALLBACK_TIME
+	var/playtime = user.client.get_exp_living(pure_numeric = TRUE) //Pure numeric, so any values returned by this proc will be in minutes.
 
-	if((intern_threshold >= playtime) && (user.mind?.assigned_role.job_flags & JOB_CAN_BE_INTERN))
+	// intern_threshold is in HOURS, while playtime is in MINUTES, so declare the appropriate defines here to allow for proper comparison.
+	if((intern_threshold >= (playtime MINUTES)) && (user.mind?.assigned_role.job_flags & JOB_CAN_BE_INTERN))
```

Comment:
The comment states that "intern_threshold is in HOURS" but I think that's incorrect?

Reply:
```dm
var/intern_threshold = (CONFIG_GET(number/use_low_living_hour_intern_hours) HOURS) || (CONFIG_GET(number/use_exp_restrictions_heads_hours) HOURS) || INTERN_THRESHOLD_FALLBACK_TIME
```

intern_threshold will pass via the decisecond define for HOURS (to respect existing config). So, it should be a correct comment? It's meant to cross-compare at the decisecond level though with the changes I'm passing through. I'll see if I can fix the comments.